sympy/physics/units/systems/mks.py

```diff
@@ -15,19 +15,30 @@
 dims = (velocity, acceleration, momentum, force, energy, power, pressure,
         frequency, action)
 
-units = [m, g, s, J, N, W, Pa, Hz]
+units = [meter, gram, second, joule, newton, watt, pascal, hertz]
 all_units = []
 
-# Prefixes of units like g, J, N etc get added using `prefix_unit`
+# Prefixes of units like gram, joule, newton etc get added using `prefix_unit`
 # in the for loop, but the actual units have to be added manually.
-all_units.extend([g, J, N, W, Pa, Hz])
+all_units.extend([gram, joule, newton, watt, pascal, hertz])
 
 for u in units:
     all_units.extend(prefix_unit(u, PREFIXES))
-all_units.extend([G, c])
+all_units.extend([gravitational_constant, speed_of_light])
 
 # unit system
-MKS = UnitSystem(base_units=(m, kg, s), units=all_units, name="MKS", dimension_system=dimsys_length_weight_time)
+MKS = UnitSystem(base_units=(meter, kilogram, second), units=all_units, name="MKS", dimension_system=dimsys_length_weight_time, preferred_units={
```

Comment:
what about calling the dict _derived units_ instead of _preferred units_? Indeed, they are the [officially derived units](https://en.wikipedia.org/wiki/SI_derived_unit).

Reply:
Sounds good, will do.